.github/workflows/turbo.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 2
-      - uses: actions/setup-node@v3
+      - uses: borales/actions-yarn@v4
```

Comment:
`actions/setup-node` includes yarn - is the officially managed node version manager by github, so would go with the original imho

https://github.com/actions/setup-node#caching-global-packages-data